### PR TITLE
tweak windows build scripts for GUI directory changes

### PR DESCRIPTION
**Pull request by zanderz**
@chrisnojima Assuming those symlinks are going away

### CODE REVIEW

File: packaging/windows/Jenkinsfile.groovy [hunk 1]
```diff
@@ -40,12 +40,6 @@ def publish(bucket, excluded, source) {
 
 def doBuild() {
     stage('Checkout Client') {
-        // Reset symlink due to node/git/windows problems
-        retry(3) {
-            bat 'if EXIST src\\github.com\\keybase\\client\\shared cd src\\github.com\\keybase\\client && git checkout shared'
-            bat 'if EXIST src\\github.com\\keybase\\client\\desktop\\shared cd src\\github.com\\keybase\\client && git checkout desktop/shared'
-            bat 'if EXIST src\\github.com\\keybase\\client\\desktop\\renderer\\fonts cd src\\github.com\\keybase\\client && git checkout desktop/renderer/fonts'
```
Comment (zanderz) [inline packaging/windows/Jenkinsfile.groovy]: ok maybe we need the fonts one still?

**Comment by chrisnojima** [comment]
i actually updated my PR again to remove the symlinks wanna test windows again

**Comment by zanderz** [comment]
It built, thanks. But if you wouldn't mind this change, it will make the Jenkins production build work with these dir changes.

File: packaging/windows/buildui.bat [hunk 2]
```diff
@@ -7,7 +7,7 @@ echo on
 call "%ProgramFiles(x86)%\\Microsoft Visual Studio 14.0\\vc\\bin\\vcvars32.bat"
 
 echo on
-pushd  %GOPATH%\src\github.com\keybase\client\desktop
+pushd  %GOPATH%\src\github.com\keybase\client\shared
 echo Calling npm i
 call npm i
```
Comment (chrisnojima) [inline packaging/windows/buildui.bat]: we should move npm to yarn here also 
```npm i``` => ```yarn install```
```npm run blah``` => ```yarn run blah```

**Pull request merged**